### ISSUE TITLE
Extra-abdominal umbilical vein varix

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7022]
**Preferred term label:** Extra- abdominal umbilical cord varix

**Synonyms**

**Definition (free text, please give PubMed ID)**  Focal dilatation of the extra-abdominal portion of the umbilical vein. An extra-abdominal umbilical varix is difficult to diagnose prenatally as it can appear like a cyst on ultrasound prior to birth.   PMID: 31061189
** Comment: Classification of an intra-abdominal umbilical vein varix includes measurement of an umbilical vein diameter ≥9 mm or wider than 50% the diameter of the intrahepatic umbilical vein. There is not clear data on classifying measurements of an extra-abdominal varix. 

**Parent term (use hpo.jax.org/app)**
Umbilical vein varix HP:0030656
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fetal anemia  PMID: 33099808

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added, thanks @patt1morgan
